### DISCUSSION TITLE
QR(Quick Response) Code based citizen feedback system for Police station

### DISCUSSION THREAD

**Discussion by URVESH1121** [scaleracademy/open-source-projects#119] [Ideas]
## 🢂 This project will give solutions to citizens to provide feedback by just scanning a QR code pasted in Police Station for feedback. Citizens will be required to fill out a feedback form online. Any officer with login rights can directly monitor the perception of the public regarding policing in Police Station/Sub Division/District/cities.

**➤ AIM:-** **`The application calculates the perception of the public about the Police.`**

**➤ Flow Example:-** Respondent Scans the QR Code -> Login screen opens on Respondent mobile -> OTP is sent on respondent mobile - > Respondent fills the Feedback form -> Respondent submits feedback.

➯The QR code pasted will also have instructions/guides regarding the steps. Viz. open the camera, select the type of phone (iPhone/Android) user, and Tap the notification for further action on the Web-based feedback form.

**NOTE:-** There will be no installation of any App on citizen's mobile phones.

➤ `Feedback will be taken on the following questions:`

1. How did you come to the police station?
     a. Through a person known to a police officer
     b. With a neighbor/ local leader
     c. On your own

2. After how much time you were heard in PS
      a. More than 15 minutes
      b. 15 minutes
      c. 10 minutes
      d. 5 minutes
      e. Immediately

3. How would you describe your experience with police officers in the police station? (__) Maximum 300 characters allowed.

4. Submit

➯After the submission, there is information about the citizen service app and the contact number of the police control room.
    
**`Admin Login`**
```
1. Admin Login Screen
2. Admin Overview
3. Admin Detailed
4. Generate QR code
```
→There shall be a website for accessing the feedback management system. Login will be via email id and password. Captcha will be 
included.
 
**`The reports will be`**
```
1. Number of feedback
2. Total Feedback
3. Police station wise
4. Sub division-wide
6. District wise
7. Responses
```

➯This will be shown in a bar/pie/histogram color chart format. The data should be able to be viewed in tabular format and also can be downloaded in excel format.

**Comment by akashhapani** [comment]
Very good ideas

**Comment by 21pravi** [comment]
nice idea

**Comment by nenisrudani7** [comment]
code?

**Comment by Jeeltechnocomet28** [comment]
Sir we are working on same project Feedback Form Nu Example: [https://demo-site-view.netlify.app/](url) and je bhi feedback malse enu page [https://musicalnight2023.netlify.app/](url) aa rite hse, 

Jo koi ne pn suggestion hoy to plzz support karjo 

Contact : [https://jeelkarkar.netlify.app/](url)
